content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -0,0 +1,85 @@
+---
+linktitle: Override a Prefab
+title: Override a Prefab
+description: Learn how to override a prefab in Open 3D Engine (O3DE).
+weight: 200
+toc: true
+---
+
+We learned in the previous tutorial that when a prefab is open in [Prefab Edit Mode](/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics#edit-a-prefab), changes to its content are automatically propagated to all instances linked to that prefab. Sometimes though, it is useful to alter a prefab instance on its own without affecting the other instances. For example, a level may have multiple instances of the same car prefab, but each car can be a different color.
+
+If just the prefab that's in Prefab Edit mode can expose its content for editing while its nested prefab instances remain closed, how can we isolate changes to a single instance?
+
+The editor provides an **Override UX** mode that allows the content of prefab instances under the prefab that's open for editing to be exposed.
+
+In the below image, the Level is in Prefab Edit mode while the content of the Car prefab instances is expanded and accessible for editing:
+
+![Level in Prefab Edit Mode in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/level-prefab-edit.png)
+
+Now when a change is made to a prefab instance, it will be registered as an override and isolated to that single instance. The level itself is responsible for storing the override information of its nested prefabs.
```

Comment:
_.... when a change is made to a **car** prefab instance, ...._

Adding the 'car' instance would be good to avoid any confusion